elide-standalone/src/main/java/com/yahoo/elide/standalone/config/ElideResourceConfig.java

```
@@ -110,27 +134,34 @@ protected void configure() {
             @Override
             protected void configure() {
                 Map<String, Swagger> swaggerDocs = settings.enableSwagger();
                 if (!swaggerDocs.isEmpty()) {
                     // Include the async models in swagger docs
-                    if(settings.enableAsync()) {
-                        EntityDictionary dictionary = new EntityDictionary(new HashMap());
+                    if (settings.enableAsync()) {
+                        EntityDictionary dictionary = new EntityDictionary(new HashMap<>());
                         dictionary.bindEntity(AsyncQuery.class);
                         dictionary.bindEntity(AsyncQueryResult.class);
-                         
-                        Info info = new Info().title("Async Service").version("1.0");
-
-                        SwaggerBuilder builder = new SwaggerBuilder(dictionary, info);
-                        
-                        //Default value of getJsonApiPathSpec() ends with /* at the end. need to remove.
-                        String asyncBasePath = settings.getJsonApiPathSpec().replaceAll("/\\*", "");
-
-                        Swagger swagger = builder.build().basePath(asyncBasePath);
-
+                        Swagger swagger = setupSwagger(dictionary, settings.getJsonApiPathSpec(), "Async Service");
                         swaggerDocs.put("async", swagger);
                     }
+                    // bind dynamic models
+                    if (settings.enableDynamicModelConfig()) {
+                        EntityDictionary dictionary = new EntityDictionary(new HashMap<>());
+                        try {
+                            for (Class entity : Util.dynamicEntityCompiler.findAnnotatedClasses(Entity.class)) {
+                                dictionary.bindEntity(entity);
+                            }
+                        } catch (ClassNotFoundException e) {
+                            log.error("error while binding class");
```

We can at least log the class name since we are swallowing the exception.